writeups/spacesos/README.md

```diff
@@ -0,0 +1,40 @@
+# Spacesos Writeup
+Есть сервис на дарте, который через `messages.dart` отправляет сообщения на сервис на php. В нём и будут все уязвимости.
+## First vulnerability
+Как мы получаем краши пользователя:
+```dart
+final resp = await http.get("${_baseUrl}/recv/?user=${user}");
+```
+```php 
+recv.php
+<?php  
+  $user = $_GET['user'];  
+  
+  chdir("/tmp/messages");  
+ echo file_get_contents(path_normalize($user)) ?? "";
+```
+Если в имени user будет #, то в `$_GET['user']` вернёт имя без #
+Получается можно посмотреть все public crashes, нарегать юзеров с таким же именем и #xxxx на конце и мы получим краши юзеров без #xxxx
```

Comment:
The reader can misunderstood the sentence like 'the vuln allow you to read public crashes'